src/transformers/models/gpt_oss/modeling_gpt_oss.py

```diff
@@ -71,10 +71,10 @@ def __init__(self, config):
         self.num_experts = config.num_local_experts
         self.hidden_size = config.hidden_size
         self.expert_dim = self.intermediate_size
-        self.gate_up_proj = nn.Parameter(torch.empty(self.num_experts, self.hidden_size, 2 * self.expert_dim))
-        self.gate_up_proj_bias = nn.Parameter(torch.empty(self.num_experts, 2 * self.expert_dim))
+        self.gate_up_proj = nn.Parameter(torch.zeros(self.num_experts, self.hidden_size, 2 * self.expert_dim))
```

Comment:
Curious why these are zeros and not empty?

Reply:
This will be updated I think, zeros is not good it allocates memory